cmd/ci-secret-bootstrap/main.go

```diff
@@ -59,6 +64,7 @@ func parseOptions() options {
 	fs.BoolVar(&o.force, "force", false, "If true, update the secrets even if existing one differs from Bitwarden items instead of existing with error. Default false.")
 	fs.StringVar(&o.logLevel, "log-level", "info", fmt.Sprintf("Log level is one of %v.", logrus.AllLevels))
 	fs.StringVar(&o.impersonateUser, "as", "", "Username to impersonate")
+	fs.IntVar(&o.maxConcurrency, "concurrency", 25, "Maximum number of concurrent in-flight goroutines to BitWarden.")
```

Comment:
So, why is 25 a good default?

Reply:
I would be happy with GOMAXPROCS

Reply:
Blind copying from 

https://github.com/kubernetes/test-infra/blob/e41d9b64ebfd3db18ed9075eb2b7ec796fb215f9/ghproxy/ghproxy.go#L121

May plan is to try to increase the number if 25 works and see if it gets any faster.
Decrease to 1 if 25 does not work. 1 will tell us if it is the concurrent problem at all.


Do you have a better number in mind?

Reply:
Fine to keep this

Reply:
Took `GOMAXPROCS `.